app/src/main/java/com/example/flixster/RegisterActivity.java

```diff
@@ -0,0 +1,81 @@
+package com.example.flixster;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.graphics.drawable.ColorDrawable;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.example.flixster.databinding.ActivityLogInBinding;
+import com.example.flixster.databinding.ActivityRegisterBinding;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+
+public class RegisterActivity extends AppCompatActivity {
+
+    private EditText name;
+    private EditText date;
+    private EditText etEmail;
+    private EditText etPassword;
+    private Button register;
+    private TextView login;
+
+    String email;
+    String password;
+    FirebaseAuth fAuth;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+
+        super.onCreate(savedInstanceState);
+        ActivityRegisterBinding binding = ActivityRegisterBinding.inflate(getLayoutInflater());
+        setContentView(binding.getRoot());
+        getSupportActionBar().setBackgroundDrawable(new ColorDrawable(getResources().getColor(R.color.purple)));
```

Comment:
Same as the previous activities, I think you can also change the color in the styles.xml resource file